client/Models.js

```diff
@@ -43,6 +43,7 @@ export class Operation {
         this.date        = has(arg, 'date') && new Date(arg.date);
         this.amount      = has(arg, 'amount') && arg.amount;
         this.binary      = (maybeHas(arg, 'binary') && arg.binary) || null;
+        this.appDetails  = (maybeHas(arg, 'appDetails') && arg.appDetails) || null;
```

Comment:
So, did you abandon the initial plan to merge `this.binary` and `this.appDetails` into a single field? That's a bit more work, but that's more interesting ;)